### ISSUE TITLE
Add a context explaination 

### ISSUE THREAD

**Issue by K-PANIK** [YansKhoja/TransfertStyle#9]
https://github.com/YansKhoja/TransfertStyle/blob/e167d8f339813bca43bfe9c34d9b4207e0075de1/README.md?plain=1#L1

Vvg19. It is a neutoral network already trained for Image Recognition from the ILSVR challenge: ImageNet Large Scale Visual Recognition Challenge (ILSVRC) evaluates algorithms for object detection and image classification at large scale...

vvg19 is a variant of VGG model which in short consists of 19 layers (16 convolution layers, 3 Fully connected layer, 5 MaxPool layers and 1 SoftMax layer). There are other variants of VGG like VGG11, VGG16 and others. VGG19 has 19.6 billion FLOPs.

From an already trained neuronal network (in your case vvg19), you have selected outputs at specifics layers to compute an image transfert style depending on introduced hyperparamters. The main purpose of your report is your feedback explaination of hyperparamters and an empirically defined method to chose them.

Your training function consume GPU processing. But what is your consumption rate ? Flops?

**In fact you have used a pretrained neural network layers such as using a simple program library. It's engenious.**. 

Could you please explain how you have choose this specifics layers? What criteria are need to select a layer? How do you know that a layer is style responsible?
What is the format of the layers database ? What data volume It consist of?

Congratulations for work you've done. Thank you for sharing.
